pkg/products/grafana/prometheusRules.go

```diff
@@ -60,14 +61,22 @@ func (r *Reconciler) newAlertReconciler(logger l.Logger, installType string, nam
 }
 
 func (r *Reconciler) removeGrafanaOperatorAlerts(installType string, ctx context.Context, apiClient k8sclient.Client) error {
-	oboNamespace := "redhat-" + resources.InstallationNames[installType] + "-operator-observability"
+	installationName := resources.InstallationNames[installType]
```

Comment:
This looks a bit overcomplicated, WDYT if in removeGrafanaOperatorAlerts we would accept "nsPrefix" string, and where the func is called we can get the nsPrefix from "installation.Spec.NamespacePrefix" then annotate the nsPrefix with "-operator-observability"

Reply:
sure, thanks a lot for this comment.  Updated.